test/unit/specs/components/Personalization/createApplyPropositions.spec.js

```diff
@@ -235,7 +235,7 @@ describe("Personalization::createApplyPropositions", () => {
       let numReturnedPropositions = 0;
       expect(originalPropositions).toEqual(MIXED_PROPOSITIONS);
       result.propositions.forEach(proposition => {
-        const original = originalPropositions.find(
+        const original = originalPropositions.filter(
```

Comment:
This should be `const [original] = originalPropositions.filter(…)` because `filter()` returns an array of matching elements, where `find()` returns a single matching element. As written, the assertion on line 243 will always pass because an instance of a proposition will never be the same as an array containing a single instance of a proposition.